src/utils/curriculum/isVisibleUnit.test.ts

```diff
@@ -0,0 +1,10 @@
+// import { isVisibleUnit } from "./isVisibleUnit.test";
+
+// import { YearSelection } from "@/components/CurriculumComponents/CurriculumVisualiser";
+// import { CurriculumUnitsTabData } from "@/node-lib/curriculum-api-2023";
+
+// TODO: Add a unit test for isVisibleUnit utility function
+
+describe("isVisibleUnit", () => {
+  it.todo("This function needs to be tested");
```

Comment:
Can you add a ticket to notion for this, and mark it as tech debt and add a comment above this line.

Reply:
Done